app/routes/($locale).account.edit.tsx

```diff
@@ -10,9 +10,9 @@ import type {
 } from "@shopify/hydrogen/customer-account-api-types";
 import { type ActionFunction, json, redirect } from "@shopify/remix-oxygen";
 import invariant from "tiny-invariant";
+import Button from "~/components/button";
```

Comment:
@viethung26 can you use button from `"~/components/button"` instead of from `"~/modules/button"`. I'm gonna remove some ui components from the modules dir in the future